custom_components/tuya_local/devices/kogan_air_purifier

```diff
@@ -0,0 +1,94 @@
+name: Kogan Air Purifier
+products:
+  - id: 5cum0gfsx8rhpvvl
+    name: Kogan Air Purifier KAIRPRFR2SA
+primary_entity:
+  entity: fan
+  translation_key: fan_with_presets
+  dps:
+    - id: 1
+      type: boolean
+      name: switch
+    - id: 4
+      type: string
+      name: preset_mode
+      mapping:
+        - dps_val: 1
+          value: "Sleep"
+        - dps_val: 2
+          value: "High"
+        - dps_val: 3
+          value: "Auto"
+secondary_entities:
+  - entity: sensor
+    name: Filter Remaining
```

Comment:
HA convention is that only the first word in a name should be capitalised. (Same for Filter reset, Timer remaining, and Air quality below)